File: core/src/org/tetawex/ecf/util/Packer.java
package org.tetawex.ecf.util;

import com.badlogic.gdx.graphics.Texture;
import com.badlogic.gdx.tools.texturepacker.TexturePacker;

public class Packer {
    public static void main(String[] args) throws Exception {
        TexturePacker.Settings settings = new TexturePacker.Settings();
        settings.filterMin = Texture.TextureFilter.Linear;
        settings.filterMag = Texture.TextureFilter.Linear;
        TexturePacker.process(settings, "assets/textures", "assets", "atlas");
    }
}

File: core/src/org/tetawex/ecf/util/Packer.kt
package org.tetawex.ecf.util

import com.badlogic.gdx.graphics.Texture
import com.badlogic.gdx.tools.texturepacker.TexturePacker

object Packer {
    @Throws(Exception::class)
    @JvmStatic
    fun main(args: Array<String>) {
        val settings = TexturePacker.Settings()
        settings.filterMin = Texture.TextureFilter.Linear
        settings.filterMag = Texture.TextureFilter.Linear
        TexturePacker.process(settings, "assets/textures", "assets", "atlas")
    }
}
